tests/data_binary_classification/Readme.md

```diff
@@ -0,0 +1,11 @@
+### Readme test data - binary classification
+This directory contains data that can be used to test aspects binary classification. Two different labels are annotated: **Form** and **Farbe**.
+There are three files containing training, test and validation data:
+- test.txt (20 instances)
+- train.txt (70 instances)
+- val.txt (10 instances)
+Each dataset contains three columns:
```

Comment:
where did you get the data from?

Reply:
the data was created by hand. shall I add something about that to the readme?

Reply:
maybe just for the sake of clarity. In the beginning, I didn't know but then when I looked through the examples it became quite obvious. 
some of my favourites:
_Ihr kreisrundes Gesicht ist schön._
_Wir rollen einen runden Klumpen._
_Er trägt einen spitzen Mund._
_Wir laufen auf einer ovalen Rutsche._